rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/ReleaseController.java

```diff
@@ -1834,4 +1834,58 @@ public ResponseEntity<List<Map<String,String>>> getReleaseLicenseInfo(
         }
         return new ResponseEntity<>(results, HttpStatus.OK);
     }
+
+    @Operation(
+            summary = "Get license data.",
+            description = "Get license source files list and other license data from release attachment.",
+            tags = {"Releases"},
+            responses = {
+                    @ApiResponse(
+                            responseCode = "200", description = "License Data for the attachment.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            schema = @Schema(
+                                                    example = """
+                                                        {
+                                                          "relId": "54321",
+                                                          "relName": "releaseName",
+                                                          "data": [
+                                                            {
+                                                              "licName": "MIT",
+                                                              "licType": "Global",
+                                                              "srcFiles": "srcFileList",
```

Comment:
`srcFiles` should look like:
```json
{
  "srcFiles": [
    "dir1/file1",
    "dir2/file2"
  ]
}
```

They currently look like:
```json
{
  "srcFiles": "[dir1/file1\n,dir2/file2]"
}
```

This is bad for REST API and must be fixed.